reflex/constants.py

```diff
@@ -42,13 +44,14 @@ def get_value(key: str, default: Any = None, type_: Type = str) -> Type:
 
 # App names and versions.
 # The name of the Reflex package.
-MODULE_NAME = "reflex"
+MODULE_NAME = "reflex" if IS_WINDOWS else ".reflex"
 # The current version of Reflex.
 VERSION = metadata.version(MODULE_NAME)
 
 # Files and directories used to init a new project.
+HOME_VAR = "%USERPROFILE%\\AppData\\Roaming" if IS_WINDOWS else "$HOME"
 # The directory to store reflex dependencies.
-REFLEX_DIR = os.path.expandvars(os.path.join("$HOME", f".{MODULE_NAME}"))
+REFLEX_DIR = os.path.expandvars(os.path.join(HOME_VAR, MODULE_NAME))
```

Comment:
```suggestion
REFLEX_DIR = os.path.expandvars(
    os.path.join(
        HOME_VAR,
        ("." if not IS_WINDOWS else "") + MODULE_NAME,
    ),
)
```

we shouldn't be changing `MODULE_NAME`, since that will break `VERSION = metadata.version(MODULE_NAME)`

ultimately we should refactor `REFLEX_DIR` to use [platformdirs](https://pypi.org/project/platformdirs/) eventually